settings.gradle

```diff
@@ -1,2 +1,2 @@
 include "wherehows-common", "hadoop-dataset-extractor-standalone", "metadata-etl", "data-model", "backend-service",
-  "web"
+  "web", "RestLiClient"
```

Comment:
Is it too late the change the module name? This "RestLiClient" may not conform to the module naming convention.